Tools/i18n/msgfmt.py

```diff
@@ -98,18 +102,44 @@ def generate():
     return output
 
 
-def make(filename, outfile):
-    ID = 1
-    STR = 2
-    CTXT = 3
+def make(filenames, outfile):
```

Comment:
Feel free to ignore, but you could even do
```suggestion
def make(filenames, outfile=None):
```
and then you can call `make(filename)` instead of `make(filename, None)` on line 281.